search/search_quality_tests/search_quality_tests.cpp

```diff
@@ -60,6 +68,14 @@ map<string, m2::RectD> const kViewports = {
 string const kDefaultQueriesPathSuffix = "/../search/search_quality_tests/queries.txt";
 string const kEmptyResult = "<empty>";
 
+void ChangeMaxNumberOfOpenFiles(size_t n)
+{
+  struct rlimit rlp;
+  getrlimit(RLIMIT_NOFILE, &rlp);
```

Comment:
If you exclude it from windows builds in pro file or in #ifdef OMIM_OS_WINDOWS, then it’s ok.

> On 25.2.2016, at 23.34, ygorshenin <notifications@github.com> wrote:
> 
> In search/search_quality_tests/search_quality_tests.cpp <https://github.com/mapsme/omim/pull/2005#discussion_r54159756>:
> 
> > @@ -60,6 +68,14 @@ map<string, m2::RectD> const kViewports = {
> >  string const kDefaultQueriesPathSuffix = "/../search/search_quality_tests/queries.txt";
> >  string const kEmptyResult = "<empty>";
> >  
> > +void ChangeMaxNumberOfOpenFiles(size_t n)
> > +{
> > +  struct rlimit rlp;
> > +  getrlimit(RLIMIT_NOFILE, &rlp);
> So what? We are not planning to use this tool on Windows.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2005/files#r54159756>.
>